omod/src/main/java/org/openmrs/module/idgen/rest/resource/LogEntryResource.java

```diff
@@ -96,11 +96,6 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return description;
 	}
 
-	@PropertyGetter("source")
-	public String getSource(LogEntry logentry) {
-		return logentry.getSource().getUuid();
-	}
-
```

Comment:
Since this change is not part of the work on this issue, u can eliminate the file from your commit

Reply:
removed